core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JmxMetricPerformanceCounter.java

```diff
@@ -46,11 +48,15 @@ public JmxMetricPerformanceCounter(String id, String objectName, Collection<JmxA
     protected void send(TelemetryClient telemetryClient, String displayName, double value) {
         logger.trace("Metric JMX: {}, {}", displayName, value);
 
-    MetricTelemetry telemetry = new MetricTelemetry();
-    telemetry.markAsCustomPerfCounter();
+        MetricTelemetry telemetry = new MetricTelemetry();
         telemetry.setName(displayName);
         telemetry.setValue(value);
-        telemetry.getProperties().put("CustomPerfCounter", "true");
+
+        // keep "Available Bytes" under PerformanceCounter to prevent breaking performance blade on the ApplicationInsights Portal
+        if (displayName.equals(TOTAL_MEMORY_PC_COUNTER_NAME)) {
+            telemetry.markAsCustomPerfCounter();
+        }
```

Comment:
`TOTAL_MEMORY_PC_COUNTER_NAME` looks like it's captured in `FreeMemoryPerformanceCounter`, not here?

Reply:
TelemetryConfigurationFactory.java: line 357.  it's eventually JmxMetricPerformanceCounter.

if (PerformanceCounterContainer.INSTANCE.register(new JmxMetricPerformanceCounter(entry.getKey(), entry.getKey(), entry.getValue()))) {